.github/workflows/type-checking.yml

```diff
@@ -0,0 +1,17 @@
+name: type-checking
+on:
+  pull_request:
+
+jobs:
+  typing:
+    name: "mypy typing"
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Make environment
+        run: make install-python-dependencies
+
+      - name: Run strict mypy
+        run: mypy config_builder --config-file mypy.ini --strict
```

Comment:
Can't you just run make cli-typecheck ? (without the dependencies install) So we keep the command itself in one place only

Reply:
Done